raster-layer-file/README.md

```diff
@@ -0,0 +1,37 @@
+# Raster Layer File
+
+Demonstrates how to create and use a raster layer made from a local raster file.
+
+![](raster-layer-file.png)
```

Comment:
Obviously your choice, but this screenshot is a little unclear to me.

It doesn't look to be taken from an handheld/Android device and doesn't hint at being georeferenced (by lining up with a basemap of some kind). My vote would be something more like the image in the [Java sample](https://github.com/Esri/arcgis-runtime-samples-java/blob/master/src/main/java/com/esri/samples/raster/raster_layer_file/RasterLayerFile.png).